master/internal/api_config_policies.go

```diff
@@ -7,31 +7,13 @@ import (
 	"google.golang.org/protobuf/types/known/structpb"
 	"gopkg.in/yaml.v3"
 
+	"github.com/determined-ai/determined/master/internal/configpolicy"
 	"github.com/determined-ai/determined/proto/pkg/apiv1"
 )
 
-// workload types are used to distinguish configuration for experiments and NTSCs.
-const (
-	// Should not be used.
-	WorkloadTypeUnspecified = "UNKNOWN"
-	// Configuration policies for experiments.
-	WorkloadTypeExperiment = "EXPERIMENT"
-	// Configuration policies for NTSC.
-	WorkloadTypeNTSC = "NTSC"
-)
-
-func validWorkloadEnum(val string) bool {
-	switch val {
-	case WorkloadTypeExperiment, WorkloadTypeNTSC:
-		return true
-	default:
-		return false
-	}
-}
-
```

Comment:
Using the workload type defined in `model` instead and moving workload type validation to `configpolicy.utils`.